helm/hpcc/templates/sasha.yaml

```diff
@@ -2,12 +2,16 @@
 {{- if not $sashaServices.disabled -}}
 {{ range $sashaName, $_sasha := $sashaServices -}}
 {{ $sasha := ($_sasha | default dict) -}}
-{{ if not $sasha.disabled -}}
-{{ $_ := set $sasha "name" $sashaName -}}
-{{ $_ := set $sasha "access" (splitList " " (include "hpcc.getSashaServiceAccess" $sasha)) -}}
-{{ $commonCtx := dict "root" $ "me" $sasha -}}
-{{ $configSHA := include "hpcc.sashaConfigMap" $commonCtx | sha256sum -}}
-{{ $serviceName := printf "sasha-%s" $sashaName -}}
+{{- if not $sasha.disabled -}}
+{{- $_ := set $sasha "name" $sashaName -}}
+{{- $_ := set $sasha "access" (splitList " " (include "hpcc.getSashaServiceAccess" $sasha)) -}}
+{{- $secretsCategories := list "system" }}
+{{- if or (has "data" $sasha.access) (has "dalidata" $sasha.access) -}}
+{{- $secretsCategories := append $secretsCategories "storage" -}}
```

Comment:
I don't think this is going to work, because it's going to treat this $secretsCategories as scoped variable of the if block

To check, the "sasha-wu-archiver " Deployment should have storage secrets generated.

This is similar to the change needed in dali.yaml, where I avoided an if with a ternary clause:
```
{{- $sashaServiceSecretsCategories := append ((or (has "data" $sashaAccess) (has "dalidata" $sashaAccess)) | ternary (list "storage") list) "system" -}}
```

I think this would become:
```
{{- $secretsCategories := append ((or (has "data" $sasha.access) (has "dalidata" $sasha.access)) | ternary (list "storage") list) "system" -}}
```